src/ralph/assets/models/assets.py

```diff
@@ -454,6 +454,7 @@ class Asset(AdminAbsoluteUrlMixin, BaseObject):
         null=True,
         blank=True,
     )
+    start_usage = models.DateField(blank=True, null=True)
```

Comment:
On second thought I would add some help text about this field.

Reply:
Fixed.